src/site/content/en/blog/load-faster-like-proxx/index.md

```diff
@@ -73,8 +73,8 @@ When loaded over 3G, the user sees 4 seconds of white nothingness. **Over 2G the
 
 <figure class="w-figure">
   <picture>
-    <source srcset="proxx-first-render.webp" type="image/webp">
-    <img src="proxx-first-render.jpg">
+    <source srcset="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/CwGKJEpvyPw9UmvJf3su.webp" | imgix }}" type="image/webp">
+    {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/i4XeWT6YeUlcMbEiDaBO.jpg", alt="", width="800", height="450" %}
   </picture>
```

Comment:
I think we can replace this `picture` with just the Img shortcode. It looks like they're only using `source` to load a webp if the browser supports it. But imgix already does that on the fly.